src/components/ArticlePage.jsx

```diff
@@ -1,28 +1,35 @@
 import { useEffect, useState } from "react";
 import { useParams } from "react-router-dom";
 import { fetchArticle } from "../requests";
+import { Comments } from "./Comments";
 
 export function ArticlePage() {
   const { article_id } = useParams();
 
   const [articleData, setArticleData] = useState({});
+  const [isLoading, setIsLoading] = useState(true);
 
   useEffect(() => {
     fetchArticle(article_id).then(({ data: { articles } }) => {
```

Comment:
You'll probably want a catch in here soon for 404 for articles that don't exist (not right now I think its a ticket later)